### PR TITLE
FE-2454 - Added wiggle animation to Decimal component

### PR DESCRIPTION
**Pull request by mkrds**
# Description
Added wiggle animation to Decimal component which is supposed to indicate that something is wrong

# Testing Instructions
Try typing or pasting something that will lead to invalid value in `Decimal` component

### CODE REVIEW

File: src/__experimental__/components/decimal/decimal.component.js [hunk 1]
```diff
@@ -20,7 +21,8 @@ class Decimal extends React.Component {
     this.state = {
       value,
       visibleValue: this.formatValue(value),
-      isControlled
+      isControlled,
+      isWiggling: false
```
Comment (dawidzarzycki) [inline src/__experimental__/components/decimal/decimal.component.js]: What about changing the name from `isWiggling` animation to `isAnimating`? It will be more flexible for the future if anyone will want to change the animation behavior

File: src/__experimental__/components/decimal/decimal.component.js [hunk 2]
```diff
@@ -54,6 +56,10 @@ class Decimal extends React.Component {
     }
   }
 
+  startWiggleAnimation = () => this.setState({ isWiggling: true })
```
Comment (dawidzarzycki) [inline src/__experimental__/components/decimal/decimal.component.js]: According to the comment above. What about just `startAnimation` and `stopAnimation`

File: src/__experimental__/components/decimal/decimal.spec.js [hunk 3]
```diff
@@ -1499,7 +1540,7 @@ describe('Decimal', () => {
 
     it('has the correct automation selectors', () => {
       render();
-      expect(wrapper.current.childAt(0).getDOMNode().getAttribute('data-component')).toBe('decimal');
+      expect(wrapper.current.find(Textbox).prop('data-component')).toBe('decimal');
```
Comment (jamime) [inline src/__experimental__/components/decimal/decimal.spec.js]: Previously this was making an assertion on the DOM (which is what we need for automation). Now, this only check's that the Textbox component has the prop, not that the Textbox component rendered into the DOM.

Could we revert the change please?
